jenkins/debian/debos/scripts/buster-xfstests.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+#####################################################
+# This script create buster image for xfstests
+#####################################################
+
+set -e
+
+# Install required xfstests packages
+apt-get install --no-install-recommends xfslibs-dev uuid-dev libtool-bin \
```

Comment:
Please keep each package on a separate line, it makes it easier to see diffs when this changes.

Reply:
Also please follow the same approach as in `buster-v4l2.sh` by listing the packages in a variable i.e. `$BUILD_DEPS` so they can be uninstalled at the end of the script.  The build dependencies should not be needed as part of the rootfs at runtime, only to build the test suite.

Reply:
okay, will apply these changes.

Reply:
With this list - xfstests installs both dependencies for compilation and also for tests like `fio` or `e2fsprogs` etc. Will remove `lib` packages from this list and verify test results.

Reply:
Moved packages required for testing into `extra_packages` on `jenkins/buster-xfstests.jpl` . Also removed packages after compilation from rootfs.